app/assets/master.css

```diff
@@ -0,0 +1,73 @@
+html, body, p, th, td, li, dd, dt, pre {
+ font: 1em Verdana, Arial, Helvetica, sans-serif;
+ }
+
+div.login_bar {
+ z-index: 99999;
+ float:right; font-size:0.9em; border:1px;
+ line-height: 50px;	//size of the google+ user image
+ vertical-align:top;
+}
+
+.error { color:#FF0000; }
```

Comment:
yes that makes sense. would it be a good idea to do that as a seperate PR and pull it in at cloudformation, or to do it in a pre-commit hook?